spec/models/album_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+describe Album do
+  let(:blanks) { ["", nil] }
+
+  it { should have_valid(:name).when("a", "Album Name", "32") }
+  it { should_not have_valid(:name).when(*blanks) }
+
+  it {
+    should have_valid(:description).when("a", "Writing all the things", "32")
+  }
+  it { should_not have_valid(:name).when(*blanks) }
+
+  it "must have at least one photo" do
+    album = Album.new
+    album.save
+    expect(album.errors[:photos]).to include("You must attach at least one photo")
+  end
+
+  it "deletes its photos when it is destroyed" do
+    album = create(:album)
+    expect{album.destroy}.to change(Photo, :count).by(-2)
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.